ticdc: update the docs about create changefeed (#3985)
pingcap/docs: ticdc/manage-ticdc.md

- `--changefeed-id`: The ID of the replication task. The format must match the `^[a-zA-Z0-9]+(\-[a-zA-Z0-9]+)*$` regular expression. If this ID is not specified, TiCDC automatically generates a UUID (the version 4 format) as the ID.
- `--sink-uri`: The downstream address of the replication task. Configure `--sink-uri` according to the following format. Currently, the scheme supports `mysql`/`tidb`/`kafka`/`pulsar`.

> - `--start-ts`: Specifies the start TSO for changefeed. The TiCDC cluster starts pulling data from this TSO. The default is the current time.

Reviewer: ```suggestion
- `--start-ts`: Specifies the starting TSO of the `changefeed`. From this TSO, the TiCDC cluster starts pulling data. The default value is the current time.
```

> - `--target-ts`: Specifies the target TSO for changefeed. The TiCDC cluster pulls data until this TSO stops. The default is empty, that is, TiCDC does not stop automatically.

Reviewer: ```suggestion
- `--target-ts`: Specifies the ending TSO of the `changefeed`. To this TSO, the TiCDC cluster stops pulling data. The default value is empty, which means that TiCDC does not automatically stop pulling data.
```

> - `--config`: Specifies the configuration file for changefeed.

Reviewer: ```suggestion
- `--config`: Specifies the configuration file of the `changefeed`.
```
